static/js/utils/getHHConfigAttributes.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Returns an object which contains the data-hhconfig attributes on the element specified by the selector
+ * 
+ * @example
+ * If the attributes on the element contain 'data-hhconfig-link-target="_top"', the returned
+ * object will be { link-target: "_top" }
+ *
+ * @param {string} selector
+ * @returns {Object}
+ */
+export default function getHHConfigAttributes (selector) {
```

Comment:
can we get some unit tests for this function? would make it easier to visualize as well

Reply:
switched to data attributes so unit tests are no longer necessary